### ISSUE TITLE
Add selection box for multiple starting positions

### ISSUE THREAD

**Issue by virtuald** [robotpy/pyfrc#120]
Example JSON:

```
{
  "pyfrc": {
  
    "robot": {
      "w": 2,
      "h": 3,
      
      "start_positions": [
            {"x": 2, "y": 20, "angle": 0, "name": "lower"},
            {"x": 2, "y": 10, "angle": 0, "name": "middle"},
            {"x": 2, "y": 5, "angle": 0, "name": "top"}
        ]
    }
  }
}

```

Would allow a user to select the starting position for the robot. Useful when testing various autonomous modes.

**Comment by virtuald** [comment]
Fixed in 7f0b6d1a77dee7166f704fab61f8d6f335589dc3

**Comment by twinters007** [comment]
🙏
